### PR TITLE
Add info when copy or cut file

### PR DESCRIPTION
**Pull request by robert00s**
Adding missing information after copying or cutting in FileManager.
Adding confirm when file exist in destination location.

koreader_045
koreader_044

### CODE REVIEW

File: frontend/apps/filemanager/filemanager.lua [hunk 1]
```diff
@@ -522,35 +519,66 @@ function FileManager:pasteHere(file)
...
+        if self.cutfile then
+            info_file = infoMoveFile
+        else
+            info_file = infoCopyFile
+        end
+        local basename = util.basename(self.clipboard)
+        local mode = lfs.attributes(dest .."/" .. basename, "mode")
+        if mode == "file" or mode == "directory" then
+            UIManager:show(ConfirmBox:new {
+                text = T(_("The %1 %2 already exists. Do you want to overwrite it?"), mode, basename),
```
Comment (Frenzie) [inline frontend/apps/filemanager/filemanager.lua]: I don't think this is the best idea for localization, setting aside the fact that neither file nor directory would be translated as currently written. Redundancy is almost always better in these matters. Even just in Dutch:

>The file blabla already exists.
*Het* bestand blabla bestaat al.

>The folder/directory already exists.
*De* map/directory bestaat al.

Similar in German with different genders. Die Datei (file) and der Ordner (folder).

**Comment by KenMaltby** [comment]
Would be nice to have in the OPDS download, as well.

**Comment by Frenzie** [comment]
Thanks!

**Comment by poire-z** [comment]
@robert00s : looks like this PR reverted a bit of the code from your + button PR.
When I hit + and New Folder, enter some name, and hit Create:
`./luajit: frontend/apps/filemanager/filemanager.lua:385: attempt to call method 'createFolder' (a nil value)`
and indeed, function FileManager:createFolder() was removed by this PR.

**Comment by robert00s** [comment]
You're right. I fix it soon. Thanks.

**Pull request merged**